src/App.tsx

```diff
@@ -1,8 +1,66 @@
-import React from 'react';
+import React, { FC, useState } from 'react';
+import { getTodos, getUsers } from './Api';
+import { TodoList } from './components/TodoList/TodoList';
+
 import './App.css';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const App: FC<{}> = () => {
```

Comment:
в функциональных компонентах не нужно указывать пустые пропсы
```suggestion
const App: FC = () => {
```